openssl-sys/build/main.rs

```diff
@@ -79,6 +79,9 @@ fn main() {
     let target = env::var("TARGET").unwrap();
 
     let (lib_dirs, include_dir) = find_openssl(&target);
+    if let Some(printable_include) = include_dir.to_str() {
+        println!("cargo:rerun-if-changed={}", printable_include);
+    }
```

Comment:
I've considered doing something like this before, but I think it's a bit dangerous. With this specific setup, `include_dir` will be e.g. `/usr/include` on a system-wide install so you'd rebuild any time you updated any system header.

Reply:
I've updated the PR to use `<include>/openssl`